<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>812</width>
    <height>812</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>812</width>
    <height>812</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>812</width>
    <height>812</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>640</y>
      <width>221</width>
      <height>90</height>
     </rect>
    </property>
    <property name="text">
     <string>Сгенерировать</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>255</x>
      <y>40</y>
      <width>540</width>
      <height>540</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>255</x>
      <y>600</y>
      <width>161</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Введите текст:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>211</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Уровень коррекции:</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonLow">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Low</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonMedium">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Medium</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonQuartile">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Quartile</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonHigh">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>High</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>92</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Версия:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxMin">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>280</y>
      <width>61</width>
      <height>33</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>40</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxMax">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>320</y>
      <width>61</width>
      <height>33</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>40</number>
    </property>
    <property name="value">
     <number>40</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>111</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Минимум:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>121</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Максимум:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>360</y>
      <width>92</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Цвет:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>400</y>
      <width>92</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Фон:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>440</y>
      <width>92</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>QR-code:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditBackground">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>400</y>
      <width>101</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>#FFFFFF</string>
    </property>
    <property name="maxLength">
     <number>7</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditQRColor">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>440</y>
      <width>101</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>#000000</string>
    </property>
    <property name="maxLength">
     <number>7</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxMask">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>480</y>
      <width>61</width>
      <height>33</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>7</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>480</y>
      <width>92</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Маска:</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>520</y>
      <width>171</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Оптимальная</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>255</x>
      <y>640</y>
      <width>540</width>
      <height>90</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>812</width>
     <height>29</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string> &amp;File</string>
    </property>
    <addaction name="separator"/>
    <addaction name="save"/>
    <addaction name="separator"/>
    <addaction name="separator"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="save">
   <property name="text">
    <string>&amp;Save</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
